src/Runner.Worker/ExecutionContext.cs

```diff
@@ -550,10 +550,12 @@ public void AddIssue(Issue issue, string logMessage = null)
                 issue.Message = issue.Message[.._maxIssueMessageLength];
             }
 
+            // Tracking the line number (logFileLineNumber) and step number (stepNumber) for each issue that gets created
+            // Actions UI from the run summary page use both values to easily link to an exact locations in logs where annotations originate from
+            issue.Data["stepNumber"] = _record.Order != null ? _record.Order.ToString() : "0"; 
```

Comment:
do you want to set the `0` up or do you want to leave it empty?
ex:
```C#
if(_record.Order!=null)
{
  issue.Data["stepNumber"] = _record.Order.ToString();
}
```

Reply:
also, a small L0 might help prevent future regression. 😄

Reply:
I'm a bit indifferent but 0 should be fine considering the first `Set up job` step is always `1`. If it's 0 then we'll know that there is no Order value for the record

Reply:
if you set it to 0, it might confuse people whether the order is 0 based or 1 based.

Reply:
I've changed it to your suggestion